### PR TITLE
Handle unsupported op in nbench

### PR DESCRIPTION
**Pull request by rkimballn1**
Prints out when unsupported op is found while running models in a directory.

### CODE REVIEW

File: test/backend_test.in.cpp [hunk 1]
```diff
@@ -51,6 +51,33 @@ static const vector<element::Type> s_known_element_types = {element::from<float>
...
+    {
+        set_value_type_checked(arg->get_element_type(), arg->get_shape());
+    }
+    shared_ptr<Node> copy_with_new_args(const NodeVector& new_args) const override
+    {
+        return make_shared<UnhandledOp>(new_args[0]);
+    }
+};
+
+TEST(${BACKEND_NAME}, unhandled_op)
```
Comment (csullivan) [inline test/backend_test.in.cpp]: NGRAPH_TEST

File: src/ngraph/runtime/gpu/gpu_external_function.cpp [hunk 2]
```diff
@@ -753,6 +752,10 @@ string runtime::gpu::GPU_ExternalFunction::emit_op_as_function(const Node& node,
     // Work around a compiler warning (*node inside typeid may have effects
     // with shared pointers, which is fine here but clang doesn't like it.)
     auto handler = dispatcher.find(type_index(typeid(node)));
+    if (handler == dispatcher.end())
+    {
+        throw ngraph::unsupported_op(node->description());
```
Comment (csullivan) [inline src/ngraph/runtime/gpu/gpu_external_function.cpp]: I guess in `emit_op_as_function`, `node` is a const ref., so we need `node.description()` here.

File: test/backend_test.in.cpp [hunk 3]
```diff
@@ -51,6 +51,33 @@ static const vector<element::Type> s_known_element_types = {element::from<float>
                                                             element::from<uint32_t>(),
                                                             element::from<uint64_t>()};
 
+class UnhandledOp : public ngraph::op::Op
+{
+public:
+    UnhandledOp(const std::shared_ptr<Node>& arg)
+        : Op("Unsupported_op", {})
+    {
+        set_value_type_checked(arg->get_element_type(), arg->get_shape());
```
Comment (csullivan) [inline test/backend_test.in.cpp]: `set_value_type_checked` has been removed from `ngraph::Node`, see the new API for adding outputs to ops.

File: src/ngraph/except.hpp [hunk 4]
```diff
@@ -39,4 +40,13 @@ namespace ngraph
         {
         }
     };
+
+    class unsupported_op : public std::runtime_error
```
[the remark this answers is not in the document]
Comment (rkimballn1) [reply src/ngraph/except.hpp]: Done

**Pull request merged**